api/src/membership/membership.py

```diff
@@ -198,7 +198,7 @@ def get_access_summary(member_id: int):
     from multiaccessy.accessy import accessy_session
 
     if accessy_session is None:
-        return PreconditionFailed(f"Accessy is not configured.")
+        return dict(in_org=False, pending_invite_count=0, access_permission_group_names=[])
```

Comment:
Also, I think you should at least emit a warning here to clarify that this is an abnormal condition. It shouldn't go unnoticed.